easybuild/easyconfigs/w/wget/wget-1.17.1-foss-2016a.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'ConfigureMake'
+
+name = 'wget'
+version = '1.17.1'
+
+homepage = 'https://www.gnu.org/software/wget/'
+description = """GNU Wget is a free software package for retrieving files using HTTP, HTTPS and FTP,
+ the most widely-used Internet protocols. It is a non-interactive commandline tool,
+ so it may easily be called from scripts, cron jobs, terminals without X-Windows support, etc."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCE_TAR_XZ]
+
+dependencies = [
+    #('gnutls', '3.4.11'),   # OS dependency should be preferred if the os version is more recent then this version, it's
+    # nice to have an up to date gnutls for security reasons
+    ('PCRE', '8.38'),
+    ('libidn', '1.32'),
+    ('zlib', '1.2.8'),
+    #('OpenSSL', '1.0.1s'),  # OS dependency should be preferred if the os version is more recent then this version, it's
+    # nice to have an up to date openssl for security reasons
+]
+
+#Optionally, you can use OpenSSL insted of gnutls
+#configopts = '--with-ssl=openssl '
+#
+#osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
+osdependencies = [('gnutls')]
```

Comment:
@boegel yeah, `gnutls-devel` is not installed (remember my quattor pr?)

Don't drop `gnutls` fully, just comment it out.